frontend/components/take-away-page/component/TakeawayForm.tsx

```diff
@@ -0,0 +1,147 @@
+import { ControlledInput } from '@/components/common/ControlledInput';
+import { ControlledTestArea } from '@/components/common/ControlledTestArea';
+import { Button } from '@chakra-ui/react';
+import { useForm } from 'react-hook-form';
+import { InputsContainer } from './InputsContainer';
+import { useEffect, useState } from 'react';
+import { MenuItem } from '@/types';
+import { zodResolver } from '@hookform/resolvers/zod';
+import * as z from 'zod';
```

Comment:
can I have this change to zod. z is too confusing

Reply:
Sure! Changed